safe/report/extractors/analysis_provenance_details.py

```diff
@@ -39,51 +40,137 @@ def analysis_provenance_details_extractor(impact_report, component_metadata):
     provenance_format_args = resolve_from_dictionary(
         extra_args, 'provenance_format')
 
-    hazard_keywords = impact_report.impact_function.provenance[
-        'hazard_keywords']
+    keywords_order = [
+        'title',
+        'source',
+        'layer_purpose',
+        'layer_geometry',
+        'hazard',
+        'exposure',
+        'hazard_category',
+        'exposure_unit',
+        'value_map',
+        'value_maps',
+        'inasafe_fields',
+        'inasafe_default_values',
+        'layer_mode',
+        'hazard_layer',
+        'exposure_layer',
+        'aggregation_layer',
+        'keywords_version']
+
+    debug_mode = impact_report.impact_function.debug_mode
```

Comment:
__[flake8]__
* [F841] local variable 'debug_mode' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->